inputs/config_map.json

```diff
@@ -0,0 +1,7 @@
+{
+  "apiVersion": "v1",
+  "kind": "configMap",
```

Comment:
I think it's `ConfigMap` (capital C and M), at least according to this example: https://docs.openshift.org/latest/dev_guide/configmaps.html#overview

Also is it worth adding a `"data": {},` here? (not strictly necessary, just might make it clearer what goes where)